main.cc

```diff
@@ -1858,6 +1852,8 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             startlog.info("SSTable data integrity checker is {}.",
                     cfg->enable_sstable_data_integrity_check() ? "enabled" : "disabled");
 
+            sl_controller.local().maybe_start_update_loop(feature_service.local(), cfg);
```

Comment:
You are postponing the first reload of service levels pretty late, after the messaging service is initialized. Won't this cause problems with us not knowing the service levels of the incoming connections and using the default scheduling group for them?

Reply:
No, it shouldn't. Comment above removed call says `This starts the update loop - but no real update happens until the data accessor is not initialized.` and data accessor is initialized just after this line.

Reply:
Perhaps the name is confusing now and should be changed?

Reply:
maybe_start_update_loop().
It is confusing to call this "update loop" after this patch. It's not a loop.

Reply:
But now the main case is raft. Maybe rename it to maybe_start_legacey_update_loop()?